containers/deploy/foreman/jobs.yml

```diff
@@ -31,29 +31,21 @@
                           securityContext: {}
                           state: present
                           volumeMounts:
-                            - readOnly: true
-                              mountPath: /etc/pki/katello/certs
-                              name: certs
                             - readOnly: false
                               mountPath: /var/www/html/pub/
                               name: pub-data
                           command:
                             - ansible-playbook
                           args:
                             - build.yml
-                          image: "{{ registry }}/client-rpm:latest"
+                            - deploy.yml
+                          image: "172.30.1.1:5000/foreman/certificates:latest"
```

Comment:
Yes - since the RPM needs the certs, seemed easier to just combine all of this into generate certs, secrets and the RPM that needs them. Also good catch on me needing to build and push this to quay and update this line here (will do when we are good with the PR overall).

Reply:
We may also be able to use [ansible runner](https://github.com/ansible/ansible-runner/blob/master/Dockerfile) for this here. We can change that in a follow up if it feels like it will be a lot of work though.

Reply:
I read up a bit on `ansible-runner` and I couldn't grok the advantages over standard Ansible with the playbook command. Could you expand on how you potentially seeing it being useful here?

Reply:
I think it's mostly about not having to maintain the runtime and build of the base ansible environment.  Just trying to standardize.

Like I said, it's not super important at the moment.